cpp/tests/iterator/iterator_tests.cuh

```diff
@@ -34,8 +35,11 @@
 #include <thrust/device_vector.h>
 #include <cub/device/device_reduce.cuh>
 
-// Base Typed test fixture for iterator test
+template <typename T>
+struct IteratorTest2222 : public cudf::test::BaseFixture {
```

Comment:
What???

Reply:
Looks like this may be unused?

Reply:
Let me yank that, I thought I removed all the fake types.

I was making sure that `ccache` wasn't being used when running the compile time benchmarks.